File: Source/Affixes/Speedster.cs
using static WCSharp.Api.Common;

public class Speedster : Affix
{
    private const int AFFIX_ABILITY = Constants.ABILITY_SPEEDSTER;

    public Speedster(Wolf unit) : base(unit)
    {
        Name = $"{Colors.COLOR_ORANGE}Speedster|r";
    }

    public override void Apply()
    {
        SetUnitMoveSpeed(Unit.Unit, 522);
        Unit.Unit.AddAbility(AFFIX_ABILITY);
        base.Apply();
    }

    public override void Remove()
    {
        SetUnitMoveSpeed(Unit.Unit, Unit.Unit.BaseMovementSpeed);
        Unit.Unit.RemoveAbility(AFFIX_ABILITY);
        base.Remove();
    }
}

File: src/Affixes/Speedster.ts
export class Speedster extends Affix {
    private AFFIX_ABILITY: number = Constants.ABILITY_SPEEDSTER

    public constructor(unit: Wolf) {
        super(unit)
        Name = '{Colors.COLOR_ORANGE}Speedster|r'
    }

    public override Apply() {
        SetUnitMoveSpeed(Unit.Unit, 522)
        UnitAddAbility(this.Unit.Unit, this.AFFIX_ABILITY)
        super.Apply()
    }

    public override Remove() {
        SetUnitMoveSpeed(Unit.Unit, Unit.Unit.DefaultMovementSpeed)
        UnitRemoveAbility(this.Unit.Unit, this.AFFIX_ABILITY)
        super.Remove()
    }

    public override Pause(pause: boolean) {}
}
